e2e-tests/desktop/tests/targets.spec.js

```diff
@@ -4,19 +4,228 @@
  */
 
 import { expect, test } from '../fixtures/baseTest.js';
+import * as boundaryHttp from '../../helpers/boundary-http';
```

Comment:
I renamed all the `boundary-http` methods to remove the `http` suffix as I didn't feel the consumer needed to know everytime they call each function as you know it by just importing it from the specific folder. 

I used this as an example but I prefer doing `import * as boundaryHttp` since we have both http and CLI.
I could also do the same for the CLI methods, let me know if you have any thoughts on this or prefer the suffixes @moduli.

Reply:
Alternatively we can still just call the import as `* as boundary` and only refer to the http or cli when both are getting imported since they realistically are doing nearly the same thing and just changes how it communicates with the controller

Reply:
Doesn't feel like that makes much of a difference? I'm cool with how it is now.